ray_lightning/ray_horovod.py

```diff
@@ -237,3 +195,16 @@ def root_device(self):
                 return torch.device("cuda", 0)
         else:
             return torch.device("cpu")
+
+    def set_cuda_device_if_used(self):
+        """Set the CUDA device to use for the root node."""
+        if self.use_gpu:
+            # overwrite the logger
+            gpu_available = True
+            gpu_type = " (cuda)"
+            gpu_used = True
+            rank_zero_info(
+                f"GPU available: {gpu_available}{gpu_type}, used: {gpu_used} "
+                "(Please ignore the previous info [GPU used: False]).")
```

Comment:
Can we just include this information in the string directly, rather than having them as variables? The variables are never changing.